NetKet/Machine/rbm_spin.hpp

```diff
@@ -37,12 +37,19 @@ class RbmSpin : public AbstractMachine<T> {
   // number of hidden units
```

Comment:
I think it's better to create a separate class `RbmWaveletSpin` or something similar

Reply:
The problem is that we are now in the middle of rewriting most of the `Machine` constructors and we won't support json constructors anymore in version 2.0
I think that the best would be for you to have a look at the branch v2.0 instead of the master, so that you can be sure that the changes you are doing will be compatible with the new version. The interface for machines is relatively stable.

Reply:
Ok good to know I'll check out to this branch and do a separate class.